### PR TITLE
Remove gradient background

### PR DESCRIPTION
**Pull request by martinRenou**

### CODE REVIEW

**Comment by martinRenou** [comment]
please update snapshots

File: packages/jupytercad-extension/src/mainview.tsx [hunk 1]
```diff
@@ -39,9 +39,6 @@ THREE.BufferGeometry.prototype.computeBoundsTree = computeBoundsTree;
 THREE.BufferGeometry.prototype.disposeBoundsTree = disposeBoundsTree;
 THREE.Mesh.prototype.raycast = acceleratedRaycast;
 
-const DARK_BG_COLOR = 'linear-gradient(rgb(0, 0, 42), rgb(82, 87, 110))';
-const LIGHT_BG_COLOR = 'radial-gradient(#efeded, #8f9091)';
-
 const DEFAULT_MESH_COLOR = new THREE.Color('#434442');
 const SELECTED_MESH_COLOR = new THREE.Color('#AB5118');
```
Comment (trungleduc) [inline packages/jupytercad-extension/src/mainview.tsx:43]: Could we make the mesh color a little bit brighter?

Comment (martinRenou) [reply packages/jupytercad-extension/src/mainview.tsx:43]: Yes I'm working on it 👍🏽

File: packages/jupytercad-extension/src/mainview.tsx [hunk 2]
```diff
@@ -31,19 +31,18 @@ import {
...
 // Apply the BVH extension
 THREE.BufferGeometry.prototype.computeBoundsTree = computeBoundsTree;
 THREE.BufferGeometry.prototype.disposeBoundsTree = disposeBoundsTree;
 THREE.Mesh.prototype.raycast = acceleratedRaycast;
 
-const DARK_BG_COLOR = 'linear-gradient(rgb(0, 0, 42), rgb(82, 87, 110))';
-const LIGHT_BG_COLOR = 'radial-gradient(#efeded, #8f9091)';
+const DEFAULT_MESH_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-inverse-layout-color4'));
+const DEFAULT_EDGE_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-inverse-layout-color2'));
+const SELECTED_MESH_COLOR = () => new THREE.Color(getCSSVariableColor('--jp-brand-color0'));
```
Comment (martinRenou) [inline packages/jupytercad-extension/src/mainview.tsx:44]: Not sure about these, it's probably too expensive to evaluate everytime we need  colors. I'll change it to evaluate only when the theme changes.

**Review by trungleduc — APPROVED** [review]

**Comment by trungleduc** [comment]
Thanks!

**Pull request merged**